bridges/centralized-ethereum/src/actors/dr_sender.rs

```diff
@@ -310,12 +311,11 @@ fn deserialize_and_validate_dr_bytes(
     match DataRequestOutput::from_pb_bytes(dr_bytes) {
         Err(e) => Err(DrSenderError::Deserialization { msg: e.to_string() }),
         Ok(dr_output) => {
-            let required_reward_collateral_ratio =
-                PSEUDO_CONSENSUS_CONSTANTS_WIP0022_REWARD_COLLATERAL_RATIO;
+            let mut dr_output = dr_output.clone();
```

Comment:
I think this `.clone` isn't necessary
```suggestion
            let mut dr_output = dr_output;
```